### PR TITLE
Update: API3 under Oracles section

### PR DESCRIPTION
**Pull request by vanshwassan**
Updated content for API3 dev docs under `base-docs/docs/tools/oracles.md`

### CODE REVIEW

File: apps/base-docs/docs/tools/oracles.md [hunk 1]
```diff
@@ -32,17 +32,24 @@ hide_table_of_contents: true
...
-- [Airnode](https://docs.api3.org/guides/airnode/calling-an-airnode/): The first-party serverless Oracle solution to bring any REST API onchain.
-- [QRNG](https://docs.api3.org/guides/qrng/): Quantum Random Number Generator for verifiable quantum RNG onchain.
+Unlike traditional data feeds, reading [API3 price feeds](https://docs.api3.org/guides/dapis/) enables dApps to auction off the right to update the price feeds to searcher bots which facilitates more efficient liquidation processes for users and LPs of DeFi money markets. The OEV recaptured is returned to the dApp.
+
+Apart from data feeds, API3 also provides [Quantum Random Number Generation](https://docs.api3.org/explore/qrng/) on Base Mainnet and Testnet. QRNG is a free-to-use service that provides quantum randomness on-chain. It is powered by [Airnode](https://docs.api3.org/reference/airnode/latest/understand/), the first-party oracle that is directly operated by the [QRNG API providers](https://docs.api3.org/reference/qrng/providers.html). Read more about QRNG [here](https://docs.api3.org/reference/qrng).
```
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:41]: s/on-chain/onchain/

File: apps/base-docs/docs/tools/oracles.md [hunk 2]
```diff
@@ -32,17 +32,24 @@ hide_table_of_contents: true
 
 ## API3
 
-[API3](https://api3.org/) is building secure first-party oracles for Base.
+The API3 Market provides access to 200+ price feeds on [Base Mainnet](https://market.api3.org/base) and [Testnet](https://market.api3.org/base-sepolia-testnet). The price feeds operate as a native push oracle and can be activated instantly via the Market UI.
```
Comment (wbnns) [inline apps/base-docs/docs/tools/oracles.md:35]: s/Testnet/Base Testnet/

**Review by wbnns — CHANGES_REQUESTED** [review]
@vanshwassan 

Thanks for this PR! Just left a few nits in-line.

**Review by zencephalon — APPROVED** [review]

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>1/2</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by wbnns — APPROVED** [review]
ACK

@vanshwassan Thanks!

**Comment by brendan-defi** [comment]
@wbnns @vanshwassan could y'all clear up the merge conflicts please? i'll take a look once resolved

**Comment by wbnns** [comment]
> @wbnns @vanshwassan could y'all clear up the merge conflicts please? i'll take a look once resolved

@brendan-defi Updated, thanks 🤝

**Review by brendan-defi — APPROVED** [review]

**Pull request merged**